docs/extend/extend-apiml/api-mediation-oidc-authentication.md

```diff
@@ -164,35 +229,32 @@ For more information about the Zowe CLI Identity Federation Plug-in, see the [RE
   
-- **`components.gateway.apiml.security.oidc.jwks.uri`**  
+- **components.gateway.apiml.security.oidc.jwks.uri**  
    Specifies the URI obtained from the authorization server's metadata where the Gateway will query for the JWK used to sign and verify the access tokens.
 
-- **`components.gateway.apiml.security.oidc.userInfo.uri`**  
+- **components.gateway.apiml.security.oidc.userInfo.uri**  
    Specifies the URI obtained from the authorization server's metadata where the Gateway queries the userInfo endpoint for access token validation. 
     To enable this property, ensure that `oidc.enabled` is set to `true` and `oidc.validationType` is set to `endpoint`.
 
-- **`components.gateway.apiml.security.oidc.jwks.uri`**  
-   Specifies the URI obtained from the authorization server's metadata where the Gateway will query for the JWK used to sign and verify the access tokens.
-
-- **`components.gateway.apiml.security.oidc.jwks.refreshInternalHours`**  
-   Specifies the frequency in hours to refresh the JWK keys from the OIDC provider. Defaults to one hour.  
+- **components.gateway.apiml.security.oidc.jwks.refreshInternalHours**  
+   (Optional) Specifies the frequency in hours to refresh the JWK keys from the OIDC provider. Defaults to one hour.  
 
-- **`components.gateway.apiml.security.oidc.identityMapperUser`**  
+- **components.gateway.apiml.security.oidc.identityMapperUser**  
     (Optional) If the userId is different from the default Zowe runtime userId (`ZWESVUSR`), specify the `identityMapperUser` userId to configure API ML access to the external user identity mapper.
 
     **Note:** User authorization is required to use the `IRR.RUSERMAP` resource within the `FACILITY` class. The default value is `ZWESVUSR`. Permissions are set up during installation with the `ZWESECUR` JCL or workflow. To authenticate to the mapping API, a JWT is sent with the request. The token represents the user that is configured with this property.
 
-- **`apiml.security.oidc.identityMapperUrl`**  
-  This property informs the Gateway about the location of this API. ZSS is the default API provider in Zowe, but if you are using Zowe release 2.14 or a later version, we recommend you use the [API ML internal mapper](../../user-guide/api-mediation/configuration-client-certificates.md#configure-internal-api-ml-mapper). You can provide your own API to perform the mapping. In this case, it is necessary to customize this value.
+- **apiml.security.oidc.identityMapperUrl**  
+  (Optional) This property informs the Gateway about the location of the identity mapper REST API. ZSS is the default API provider in Zowe, but if you are using Zowe release 2.14 or a later version, we recommend you use the [API ML internal mapper](../../user-guide/api-mediation/configuration-client-certificates.md#configure-internal-api-ml-mapper). In case the internal mapper is configured, this parameter is ignored.
 
     The following URL is the default value for Zowe and ZSS:
```

Comment:
It seems that it would be appropriate to have a full example yaml with all of these properties populated.

Reply:
I have added an example after this description. Some parameters are exclusive and I think it would confuse users to see all of them in the example.